### DISCUSSION TITLE
添加去广告的配置能力

### DISCUSSION THREAD

**Discussion by hussion** [kkkgo/PaoPaoDNS#77] [想法]
能否添加类似`force_no_resolve.txt`的配置文件或者最好是`url`，可以支持添加/导入广告的域名列表，实现简单的去广告的能力 ？

**Comment by kkkgo** [comment]
你可以套一个`AdguardHome`。
“去广告”会可能引起非预期的网页加载响应延迟、链接打不开、视频无法播放等问题。
因此DNS应该做的是尽量保证原始数据准确。去广告功能应该分离开。而`AdguardHome`也是一个功能上已经非常完善的去广告DNS服务程序，包括网页在线管理订阅URL、自定义过滤规则、解析adblock域名等，总的来说，如果要实现这些功能，就是再造一个体积翻倍的轮子。
